### ISSUE TITLE
Accessibility: Suggestions icon not visible on dark theme

### ISSUE THREAD

**Issue by jermanuts** [WeblateOrg/weblate#20834]
### Affected page or workflow

When a component has a suggestion the logo is not visible on dark theme

### Steps to reproduce

go to a component that has a suggestion

### Expected behavior

the icon should have better contrast

### Actual behavior

bad contrast

### Impact

Minor issue or inconsistent behavior

### Screenshots or recordings

<img width="453" height="571" alt="Image" src="https://github.com/user-attachments/assets/a090e4ef-e25c-4c01-bc07-27dc3afb9a13" />
